apps/api/src/app/events/usecases/trigger-event/trigger-event.usecase.ts

```diff
@@ -224,7 +224,7 @@ export class TriggerEvent {
         status: LogStatusEnum.ERROR,
         environmentId: command.environmentId,
         organizationId: command.organizationId,
-        text: 'SubscriberId missing in to property',
+        text: 'SubscriberId missing from to property',
```

Comment:
`in to` sometimes should be written `into`. But here, it's essentially inappropriate to use `in` for a thing that is missing -- such a thing is missing _from_ a place -- it isn't _in_ the place.

Reply:
Rollback this. What this means is:
```
'SubscriberId missing from `to` property',
```
It is referring to the `to` property that represents who is going to receive the notification.